Makefile

```diff
@@ -1,9 +1,12 @@
 # Makefile for Sphinx documentation
-#
+
+SHELL=bash
+.DEFAULT_GOAL = all
```

Comment:
Please remove. This prevents using just `make` on the command line to output the list of available make targets and their comments.

Reply:
Ok, I will set "help" as the default goal. The problem is that makefiles use the first defined target when you simply run `make`, so it can be confusing if people aren't familiar with makefiles.

Reply:
I think you are right. `make help` is more understandable and would be a good default target for just `make`.

We will also need to update the Contributing docs to use `make help` instead of just `make` to get the list of targets and their comments. Explicit is better than implicit.

Reply:
In the latest version of the Makefile "help" is now the default target. I think that's the best. Usually "all" is the default target in makefile, when I've set that it was by force of habit.

Reply:
Fixed in 3ee76c9f93654dbbe9505b70c6054ccf9ca30797